debian/preinst

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+set -e
+
+# Upgrade from <2.0 needs to stop the .target
```

Comment:
Nit: comments in postinst have trailing `.`s…

Reply:
This was a conscious choice to avoid abiguity on the filename.